scripts/check_imports_and_dependencies.py

```diff
@@ -90,7 +90,20 @@ def get_imports_for_file(pyfile: Union[str, Path]) -> List[str]:
         with open(pyfile, "r") as f:
             statements = f.read()
         instructions = dis.get_instructions(statements)  # type: ignore
-        imports = [i for i in instructions if "IMPORT" in i.opname]
```

Comment:
This worked until yesterday, I don't know what introduced this issue but this leads to check failing for following block of code

https://github.com/valory-xyz/open-aea/blob/fecfeae123c82ade9799e856d09268e86b772271/aea/cli/push.py#L51-L56

Which shouldn't be the case

Reply:
If you're talking about the script, No it does not raise any errors it just fails the check saying `aea_cli_ipfs` is not part of the code dependencies which is true for since the cli IPFS plugin is optional that's why we import it in a try block.

Reply:
but you would expect some sort of error to accommodate the failure, right? Unless it's something severe like a segmentation fault. So is it functional now or still throwing error, that is not clear to me. If this works with `try / except` than good lets move on